apps/frontend/src/app/Data/Characters/Nahida/index.tsx

```diff
@@ -523,6 +523,7 @@ const sheet: ICharacterSheet = {
         teamBuff: true,
         path: condC2QSAPath,
         value: condC2QSA,
+        canShow: equal(condC2Bloom, 'on', 1),
```

Comment:
Technically these are not linked, since the quicken, etal buff has an 8s duration